design-proposals/vm-delete-protection.md

```diff
@@ -0,0 +1,155 @@
+# Overview
+
+VirtualMachine Delete Protection prevents the accidental deletion of VMs from a cluster. As long as deletion protection
+is activated for certain VM objects, they cannot be deleted unless the protection is intentionally removed.
```

Comment:
Can you add a section that explains how the user is expected to remove the protection (remove the annotation)? e.g. by patch operation?

Reply:
Done.